docs/docs/develop/glob-patterns.md

```diff
@@ -32,21 +34,31 @@ glob.max_total_size: 1073741824000
 ## Extract policies
 
 Rill also supports extracting a subset of data matching a glob pattern. There are two types of extract policies:
-1. **File based limits** : These limits restrict the number of files that are ingested.
-2. **Row based limits** : These limits restrict the amount of data that is ingested from each file.
+1. **Row based limits** : These limits restrict the amount of data that is ingested from each file.
+2. **File based limits** : These limits restrict the number of files that are ingested.
+
 
 You can apply these policies individually or in combination to control the extraction process.
   - Here are the possible combination and their semantics.
-    - If both `rows` and `files` are specified, each file matching the `files` clause will be extracted according to the `rows` clause.
     - If only `rows` is specified, no limit on the number of files is applied. For example, getting a 1 GB `head` extract will download as many files as necessary.
     - If only `files` is specified, each file upto limit will be fully ingested.
+    - If both `rows` and `files` are specified, each file matching the `files` clause will be extracted according to the `rows` clause.
 
 Each policy can be configured by specifying two parameters:
-1. **Size** : The number of files/size of data to be fetched.
-2. **Strategy** : The strategy to fetch data. Currently, only **Head** (first n up to size) or **Tail** (last n up to size) is supported.
+```yaml
+size: The size of data in bytes (for rows) or number or files (for files) to fetch.
+strategy: The strategy to fetch data. Currently, only `Head` (first n up to size) or `Tail` (last n up to size) is supported.
+```
```

Comment:
It's still better to use bullets, just mean to wrap `size` and `strategy` in code tags